.gitignore

```diff
@@ -14,7 +14,7 @@ examples/**/build/*
 **/_generated/*
 **/outputs/*
 .coverage
-
+examples/multimodal_chat/resources/*
```

Comment:
maybe we should put this in a .gitignore inside the `multimodal_chat` directory to have it closer to the source? i'm open to either way. The thinking is that if examples move around, get renamed, or get removed in the future, the links in the global .gitignore might go stale, but this is less likely to occur for the .gitignore within the example proj directory